demoapp/src/main/AndroidManifest.xml

```diff
@@ -5,6 +5,7 @@
     <uses-permission android:name="android.permission.INTERNET"/>
 
     <application
+        android:debuggable="true"
```

Comment:
Adding this to the manifest [shouldn't be necessary](http://developer.android.com/tools/device.html) as long as the Gradle change you made is in place.